### ISSUE TITLE
Broken in Safari since "Correct lf.structs.Set type signature." commit

### ISSUE THREAD

**Issue by kenjitayama** [google/lovefield#62]
The Quick Start code isn't working in Mac Safari since the following commit
https://github.com/google/lovefield/commit/b054565617bcb5c78608d5d496f5cc415d9f4784

how to confirm
```
git checkout b054565617bcb5c78608d5d496f5cc415d9f4784
cd demos/todo
ln -s ../../dist/lovefield.js
vi todo.html # change script tag to reference lovefield.js in current directory
gulp debug
```

The error is:
```
[Error] TypeError: undefined is not an object (evaluating 'childNode.visited')
	（anonymous function） (lovefield.js, line 11270)
	forEach ([native code], line 0)
	checkCycleUtil_ (lovefield.js, line 11268)
	（anonymous function） (lovefield.js, line 11292)
	forEach ([native code], line 0)
	checkFkCycle_ (lovefield.js, line 11291)
	finalize_ (lovefield.js, line 11265)
	getSchema (lovefield.js, line 11301)
	connect (lovefield.js, line 11313)
	global code (todo.html, line 26)
```

Mac Safari version
8.0.7 (10600.7.12)

**Comment by arthurhsu** [comment]
Safari seems to have a different opinion to JavaScript standards. It treats undefined as an explicit parameter.

> new window.Set(undefined)
< Set
size: 1
__proto__: Set

> new window.Set()
< Set
size: 0
__proto__: Set

We'll need to workaround this in lf.structs.set code.

**Comment by kenjitayama** [comment]
Wow.. I tried that for myself too.
I know it's tough supporting Safari/Mobile Safari. Seen many other issues.
Cheers for the hard work.

**Comment by arthurhsu** [comment]
Fixed a5ebe1dc3feefebfeeb687887a717b687385444b
